Resources/Prototypes/Entities/Objects/Weapons/Bombs/plastic.yml

```diff
@@ -54,3 +54,57 @@
           states:
             enum.Trigger.TriggerVisualState.Primed: primed
             enum.Trigger.TriggerVisualState.Unprimed: complete
+
+- type: entity
+  name: Test bomb
+  description: Used to put holes in specific areas without too much extra hole. A saboteur's favorite.
+  parent: BaseItem
+  id: bombStickyTest
+  components:
+    - type: Sprite
+      sprite: Objects/Weapons/Bombs/c4.rsi
+      state: icon
+    - type: Item
+      sprite: Objects/Weapons/Bombs/c4.rsi
+      size: 10
+    - type: OnUseTimerTrigger
+      delay: 10
+      delayOptions: [10, 30, 60, 120, 300]
+      initialBeepDelay: 0
+      beepSound: /Audio/Machines/Nuke/general_beep.ogg
+      startOnStick: true
+    - type: TriggerOnSignal
+    - type: SignalReceiver
+    - type: Sticky
+      stickDelay: 0
+      unstickDelay: 0
+      stickPopupStart: comp-sticky-start-stick-bomb
+      stickPopupSuccess: comp-sticky-success-stick-bomb
+      unstickPopupStart: comp-sticky-start-unstick-bomb
+      unstickPopupSuccess: comp-sticky-success-unstick-bomb
+    - type: Explosive # Powerful explosion in a very small radius. Doesn't break underplating.
+      explosionType: DemolitionCharge
+      totalIntensity: 600
+      intensitySlope: 5
+      maxIntensity: 300
+      canCreateVacuum: false
+    - type: ExplodeOnTrigger
+    - type: Damageable
+      damageContainer: Inorganic
+    - type: Destructible
+      thresholds:
+        - trigger:
+            !type:DamageTrigger
+            damage: 50
+          behaviors:
+            - !type:DoActsBehavior
+              acts: ["Destruction"]
+            - !type:ExplodeBehavior
+    - type: StickyVisualizer
+    - type: Appearance
+      visuals:
+        - type: GenericEnumVisualizer
+          key: enum.Trigger.TriggerVisuals.VisualState
+          states:
+            enum.Trigger.TriggerVisualState.Primed: primed
+            enum.Trigger.TriggerVisualState.Unprimed: complete
```

Comment:
what is the point of this?

Reply:
This was to test out my bug fix , basically a carbon copy of C4 but with no delay to stick/unstick